### DISCUSSION TITLE
Orders v0 updateShipmentStatus operation

### DISCUSSION THREAD

**Discussion by invalidCards** [abuzuhri/Amazon-SP-API-CSharp#197] [Q&A] [answered]
Hi, is it possible to call the [updateShipmentStatus](https://developer-docs.amazon.com/sp-api/docs/orders-api-v0-reference#post-ordersv0ordersorderidshipment) operation using this package? I can't seem to find a way to use it in the code.

**Comment by J-W-Chan** [comment, marked as answer]
you can see this issue [https://github.com/amzn/selling-partner-api-docs/issues/1873](url)
